hail_scripts/v02/utils/elasticsearch_client.py

```diff
@@ -165,7 +169,7 @@ def export_table_to_elasticsearch(
 
         _meta = None
         if export_globals_to_index_meta:
-            _meta = dict(hl.eval(table.globals))
+            _meta = struct_to_dict(hl.eval(table.globals))
```

Comment:
What about `json.loads(hl.eval(hl.json(table.globals))))`? I tried to find a native hail to python object but this is the closest I could find. 

This would allow us to use other hail expressions as well.

Tested with
`json.loads(hl.eval(hl.json(hl.struct(x=2))))` --> `{'x': 2}`
and
`json.loads(hl.eval(hl.json(hl.set([2]))))` --> `[2]`

Reply:
One downside to that is how `hl.json` encodes dicts.

```
hl.eval(hl.json({"foo": 1, "bar": 2}))
--> '[{"key":"bar","value":2},{"key":"foo","value":1}]'
```

Reply:
Write but table.globals will not have dicts, right?

Reply:
The gnomAD variants Hail tables' globals contain dicts.